### PR TITLE
Added parameter validation to paramsgui and GPhL parameter queries. 

### PR DESCRIPTION
**Pull request by rhfogh**
Added parameter validation to GPhL using paramsgui for runtime parameter queries. 

NB goes with matching PR in HardwareRepository

### CODE REVIEW

**Comment by IvarsKarpics** [comment]
Just a minor pylint suggestions: https://app.codacy.com/app/mxcube/mxcube/pullRequest?prid=3513574
Otherwise I am fine with the PR.

**Comment by rhfogh** [comment]
@IvarsKarpics I am working on fixing the Codacy problems. Till soon, Rasmus

**Comment by rhfogh** [comment]
Added pylint/black fixes

File: gui/utils/paramsgui.py [hunk 1]
```diff
@@ -101,13 +146,15 @@ def get_name(self):
...
+
     def __init__(self, parent, options):
         QtImport.QWidget.__init__(self, parent)
 
         # do not allow qt to stretch us vertically
-        sp = self.sizePolicy()
-        sp.setVerData(QtImport.QSizePolicy.Fixed)
-        self.setSizePolicy(sp)
+        policy = self.sizePolicy()
+        policy.setVerData(QtImport.QSizePolicy.Fixed)
```
Comment (IvarsKarpics) [inline gui/utils/paramsgui.py]: There is no setVerData in Qt5. Instead please use setVerticalPolicy: https://doc.qt.io/qt-5/qsizepolicy.html

**Review by IvarsKarpics — CHANGES_REQUESTED** [review]

File: gui/utils/paramsgui.py (hunk 1 above)
Comment (rhfogh) [reply gui/utils/paramsgui.py]: Fixed and pushed

Comment (rhfogh) [reply gui/utils/paramsgui.py]: Thanks for checking.

**Review by IvarsKarpics — APPROVED** [review]

**Pull request merged**